redwood-treehouse/build.gradle

```diff
@@ -42,6 +42,9 @@ kotlin {
     nativeMain {
       dependsOn(hostMain)
     }
+    iosMain {
+      dependsOn(nativeMain)
+    }
```

Comment:
Was this needed? The `iosMain` source set and its dependence on `nativeMain` should already have been configured globally for all multiplatform modules.

Reply:
I had to add it to resolve some error, but doesn't look like that's the case anymore - possibly because I was actually working on a really old branch, and then rebased against latest.